src/components/card/card.scss

```diff
@@ -121,6 +121,44 @@
     margin-top: -2rem;
     margin-bottom: 2rem;
   }
+
+  .media--small {
+    @include breakpoint(sm) {
+      width: 25%;
+    }
+  }
+
+  .media--medium {
+    @include breakpoint(sm) {
+      width: 37.5%;
+    }
+  }
+
+  .media--large {
+    @include breakpoint(sm) {
+      width: 50%;
+    }
+  }
+
+  &[class*="card--layout"] {
+    .media--small {
+      @include breakpoint(sm) {
+        flex: 0 0 25%;
+      }
+    }
+
+    .media--medium {
+      @include breakpoint(sm) {
+        flex: 0 0 37.5%;
+      }
+    }
+
+    .media--large {
+      @include breakpoint(sm) {
+        flex: 0 0 50%;
+      }
+    }
+  }
```

Comment:
We have a section below for this. Can we move this there?

Reply:
Move it to `Layout left and layout right shared `?